packages/es-dev-server/test/middleware/plugin-transform.test.ts

```diff
@@ -128,12 +128,55 @@ describe('plugin-transform middleware', () => {
 
     try {
       const response = await fetch(`${host}index.html`);
-      const responseText = await response.text();
 
       expect(response.status).to.equal(200);
       expect(response.headers.get('x-foo')).to.equal('bar');
     } finally {
       server.close();
     }
   });
+
+  it('caches response bodies when possible, by default', async () => {
+    const { server } = await startServer({
+      transform(ctx) {
+        if (ctx.path === '/app.js') {
+          return { body: `${Date.now()}` };
+        }
+      },
+    });
+
+    try {
+      let response = await fetch(`${host}app.js`);
+      let timestampOne = await response.text();
+
+      response = await fetch(`${host}app.js`);
+      const timestampTwo = await response.text();
+
+      expect(timestampOne).equal(timestampTwo);
+    } finally {
+      server.close();
+    }
+  });
+
+  it('allows users to turn off caching of response body', async () => {
+    const { server } = await startServer({
+      transform(ctx) {
+        if (ctx.path === '/app.js') {
+          return { body: `${Date.now()}`, disableCache: true };
```

Comment:
Let's do `transformCache: false` to highlight what kind of cache it is.

Reply:
Sounds good :) I changed it